core/app/[locale]/(default)/login/_components/login-form.tsx

```diff
@@ -71,14 +71,8 @@ export const LoginForm = () => {
       )}
 
       {isFormInvalid && (
-        <Message
-          aria-labelledby="error-message"
```

Comment:
Should we make `aria-leballedby` a prop?

Reply:
I'm not exactly sure why we did this in just 2 messages, but not all messages. In any case, we should probably handle the labels inside the component automatically if needed?